Source/EnterpriseUserAgentFramework/AppAuthEnterpriseUserAgent.h

```diff
@@ -0,0 +1,29 @@
+/*! @file AppAuthEnterpriseUserAgent.h
+   @brief AppAuth iOS SDK
+   @copyright
+       Copyright 2020 Google Inc. All Rights Reserved.
+   @copydetails
+       Licensed under the Apache License, Version 2.0 (the "License");  
+       you may not use this file except in compliance with the License.
+       You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+       Unless required by applicable law or agreed to in writing, software
+       distributed under the License is distributed on an "AS IS" BASIS,
+       WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+       See the License for the specific language governing permissions and
+       limitations under the License.
+*/
+
+#import <Foundation/Foundation.h>
+
+//! Project version number for AppAuthFramework-iOS.
+FOUNDATION_EXPORT double AppAuthCoreVersionNumber;
```

Comment:
Should this be exporting the AppAuthCoreVersionNumber? Or AppAuthEnterpriseUserAgentVersionNumber?

Reply:
Good catch, thanks.